### PR TITLE
PHP Mode will move *.el files into lisp/ directory

### PR DESCRIPTION
**Pull request by zonuexe**
refs https://github.com/emacs-php/php-mode/issues/636

### CODE REVIEW

File: recipes/php-mode [hunk 1]
```diff
@@ -1,2 +1,2 @@
 (php-mode :repo "emacs-php/php-mode" :fetcher github
-          :files (:defaults "skeleton/*.el"))
```
Comment (zonuexe) [inline recipes/php-mode:2]: `skeleton/` directory has been removed. https://github.com/emacs-php/php-mode/pull/534

**Comment by purcell** [comment]
Okay, sure, happy to merge this, but I don't believe that moving the most important two files in the repo into one subdirectory just to make the file list one entry shorter is a big win. :-)

**Comment by zonuexe** [comment]
@purcell 
Seven files are moved instead of just two(https://github.com/emacs-php/php-mode/pull/637).  The number of files will be increased later by refactoring and adding features.

**Comment by akirak** [comment]
Even if the elisp files are moved into the subdirectory, there will still be many files at the top-level directory, which makes the readme far to reach. I think this would be an issue with the design of GitHub, so it may fix the issue, e.g. by collapsing the top-level file list in the future, even though I have never seen an announcement for it yet.

**Comment by zonuexe** [comment]
@akirak Yes, your point is correct and the file list is already collapsed in mobile browsers.

**Comment by purcell** [comment]
Not quite sure what the conclusion is here, but I'm going to merge this. :-)

**Comment by akirak** [comment]
It's as confusing as our Go To Travel campaign in Japan :)

**Pull request merged**